lib/java/com/google/android/material/materialswitch/res/values/attrs.xml

```diff
@@ -46,6 +46,8 @@
            result to valid color values. Saturate(S + D) -->
       <enum name="add" value="16" />
     </attr>
+    <!-- Size that will be applied to the thumb icon drawable. -->
```

Comment:
I know this naming pattern is used throughout this file, but it's wordy, especially for something as straightforward as size. You can replace "that will be applied to" with "of" or "used for".

Reply:
Did you forget to merge the new changes?

Reply:
No, I wanted to get all my questions answered first 😄

Reply:
Ah got it :)